### DISCUSSION TITLE
Work with Shadcn styling in Payload is nightmare

### DISCUSSION THREAD

**Discussion by BulliedBeginner** [payloadcms/payload#9636] [General]
I have spent a whole day trying to get Shadcn styling works inside the Payload CMS. To get it works, I need to use external Tailwind scoped preflight library and it does not work perfectly. Some of the styles does not work as I expected due to the Payload CMS **global** SCSS styling, especially the root font size. It affects almost every Tailwind style because tailwind use **rem** instead of em or px. Can development team solve the problem? Maybe development team can provide a Tailwind version or do not use global styling so that our custom component will not get affected.

**Comment by mikecebul** [comment]
I don't have many issues as long as I'm only adding these two out of the three to `/(payload)/custom.scss`
```css
@tailwind components;
@tailwind utilities;
```

**Reply by sones3** [reply]
These two are not enough to get shadcn running unfortunately. You need some workarounds.
